test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestSharedClassesExt.java

```diff
@@ -533,7 +533,8 @@ public void testShrcJITHForExt() {
 		}
 
 		for (int i = 0; i < lines.length; i++) {
-			if (lines[i].contains("java/lang/Object")) {
+			if (lines[i].contains("java/lang/Object") || lines[i].matches("\\s<init>\\(\\).*j9rommethod .*")) {
+				// lines[i].matches: find j9rommethod address from <init> method in openj9
```

Comment:
Please improve this comment. It should help the reader understand *why* the condition is what it is, not merely repeat it.